app/views/hackathons/digest_mailer/organizer_summary.html.erb

```diff
@@ -0,0 +1,13 @@
+<% content_for :greeting do %>
+  <p>G'day!</p>
+<% end %>
```

Comment:
If we remove this, I'll default to

```
Hey {name}
```
or
```
Hey there
```

Reply:
```suggestion
```

Reply:
that's actually sick!